### ISSUE TITLE
Upgrade to xUnit.net 2

### ISSUE THREAD

**Issue by ploeh** [GreanTech/Exude#21]
Now that xUnit.net 2 is out, we should consider upgrading Exude to xUnit.net 2.

As far as I can tell from the NuGet download statistics, it doesn't look like a lot of people are using Exude. Version 0.2.1 has 345 downloads as I'm writing this. Additionally, that version is more than a year old, so if anyone uses it for serious work, we must assume that that version works for them.

Since Semantic Versioning allows breaking changes when the major version is 0, we can upgrade to xUnit.net 2 without worrying too much about compatibility. Per the above argument, people already on Exude 0.2.1 can stay on that version if they're unable to upgrade to xUnit.net 2.

**Comment by ploeh** [comment]
@sgryt or @flensted, will one of you please add the _jump in_ tag to this issue (since I don't have the rights)?

**Comment by moodmosaic** [comment]
As it looks like, xUnit.net 2 has removed the `TestCommand` which is one of its extensibility points where Exude relies on.

We _might_ have to upgrade the target framework as well, since [xUnit.net 2 ships with many NuGet packages](http://xunit.github.io/docs/nuget-packages.html) and some of them (that we might need) target newer versions of the .NET Framework:

```
Could not install package 'xunit.extensibility.execution 2.0.0'. You are trying
to install  this package into a project that targets '.NETFramework,Version=v3.5',
but the package does not contain any assembly references or content files that
are compatible with that framework. For more information, contact the package
author.
```

**Comment by ploeh** [comment]
For the same reasons as outlined above, I don't think it'd be a problem to upgrade the target framework as well.

**Comment by moodmosaic** [comment]
The problem is the `TestCommand` which seems to me (re)moved... Searching for this in the current [xunit/src](https://github.com/xunit/xunit/tree/3fe6401d9f45de9df607c925023a3e043d6c806e/src) tree yields 0 results on my machine:

```
Searching 484 files for "TestCommand"

0 matches
```

**Comment by CumpsD** [comment]
@moodmosaic did you ever find a replacement for `TestCommand` to be able to port Exude over to netstandard + xunit2?

**Comment by moodmosaic** [comment]
@CumpsD No, I didn't find a replacement for `TestCommand`. I didn't spend much time looking at it.
